src/main/java/org/betonquest/betonquest/compatibility/worldedit/PasteSchematicEvent.java

```diff
@@ -49,6 +53,8 @@ public PasteSchematicEvent(final Instruction instruction) throws InstructionPars
         staticness = true;
         persistent = true;
         loc = instruction.getLocation();
+        rotation = new VariableNumber(BetonQuest.getInstance().getVariableProcessor(), instruction.getPackage(), instruction.getOptional("rotation", "0"));
```

Comment:
You can also use `instruction.getVarNum(instruction.getOptional("rotation", "0"))`.